ml_condor/run_condor_training_gpu.sh

```diff
@@ -10,14 +10,13 @@
 # Steps 2 and 3 are functionally identical to ml/run_training.sh
 
 set -e
-ERA=$1
+ERA_NAME=$1
 CHANNEL=$2
 TAG=$3
-folder=${ERA}_${CHANNEL}_${TAG}
-outdir=/tmp/${folder}
+NAME_USER=$4
```

Comment:
you could simply use `${USER}`, this should be your username, so you don't need to add it as an argument

Reply:
I don't believe ${USER} can be used here as this part of the code is executed in a docker image.